assets/sass/globals/forms.scss

```diff
@@ -149,6 +154,7 @@ input[type="search"] {
 .ff-textarea {
     width: 100%;
     padding: 0.5em;
+    box-sizing: border-box;
```

Comment:
Did this not already get picked up from `html {}`?

Reply:
No, but I'm not sure why. Will double check just in case we can remove this.